test/functional/admin/detailed_guides_controller_test/national_applicability_test.rb

```diff
@@ -6,6 +6,8 @@ class NationalApplicabilityTest < ActionController::TestCase
 
     setup do
       login_as create(:writer, organisation: create(:organisation))
+      current_user.permissions << "Preview design system"
```

Comment:
Would this work instead?

```suggestion
      login_as_preview_design_system_user :writer
```

Reply:
Ah, you might be right. I'd assumed `create(:writer)` would give the user an org automatically. But looking at [the factory](https://github.com/alphagov/whitehall/blob/ab98fb14d01fddd38abf79a430fb68dbcf365de8/test/factories/users.rb), I think I might be wrong on that!

No stress if it causes tests to fail – just leave it as is!

Reply:
it went 🟢  so all good. Thanks for that